sa/database.go

```diff
@@ -80,13 +71,33 @@ func NewDbMapFromConfig(config *mysql.Config, maxOpenConns int) (*gorp.DbMap, er
 		// In MariaDB, max_statement_time and long_query_time are both seconds.
 		// Note: in MySQL (which we don't use), max_statement_time is millis.
 		readTimeout := config.ReadTimeout.Seconds()
```

Comment:
I think this is actually correct - `readTimeout` is the read timeout, and the query timeout is 95% of that.